test/torchaudio_unittest/datasets/librimix_test.py

```diff
@@ -0,0 +1,93 @@
+import os
+
+from parameterized import parameterized
+
+from torchaudio.datasets import LibriMix
+from torchaudio_unittest.common_utils import get_whitenoise, normalize_wav, save_wav, TempDirMixin, TorchaudioTestCase
+
+_TASKS_TO_MIXTURE = {
+    "sep_clean": "mix_clean",
+    "enh_single": "mix_single",
+    "enh_both": "mix_both",
+    "sep_noisy": "mix_both",
+}
+
+
+def get_mock_dataset(root_dir: str, num_speaker: int):
+    """
+    root_dir: directory to the mocked dataset
+    """
+    mocked_data = []
+    sample_rate = 16000
+    seed = 0
+    base_dir = os.path.join(root_dir, f"Libri{num_speaker}Mix", "wav16k", "min", "train-360")
```

Comment:
Not sure why this does not fail on Windows. The resulting `base_dir` here should be `wav16k\min\train-360\`, while the corresponding part in Dataset implementation is `wav16k/min\train-360\`.

Reply:
I see what you mean. `wav16k/min` needs to be changed to `path.join("wav16k", "min")`

Reply:
However, that makes me wonder if the original generation script works that way on Windows. If they use slash in their script then a directory with slash in it is generated. We need to check that, but I don't have access to Windows at the moment.